### ISSUE TITLE
3d mapping but also direct control from app disruptive

### ISSUE THREAD

**Issue by bartbakels** [dr1rrb/ha-twinkly#6]
It seems when this implmentation is active, it scans the twinklys every x secs. which interferes when trying to mapp the lights via the app or direct control the light effects.

**Comment by dr1rrb** [comment]
Yeah, the Twinkly devices do not allow 2 connected clients at once (when HA connects to get the current status, it will log-out the native app, even if the native app is actively interacting with the device). 

My initial thought was that won't be too disruptive since both are able to automatically re-authenticate with the device on each call, but indeed I noticed few days ago that it was really annoying while designing effects.

The only solution I've found so far is to add on HA a "Programming mode" switch which will actually disable the HA integration for a given delay (like 30mn or 1h).

The workaround for now is simply to stop Home-Assistant while programming your Twinkly devices ... which is really annoying :/
